server/route.go

```diff
@@ -221,28 +221,11 @@ func buildRoutes(app *applicationHandler) http.Handler {
 			authRouter.Post("/logout", app.LogoutUser)
 		})
 
-		uiRouter.Route("/dashboard", func(dashboardRouter chi.Router) {
-			dashboardRouter.Use(requireGroup(app.groupRepo, app.cache))
-			dashboardRouter.Use(rateLimitByGroupID(app.limiter))
-
-			dashboardRouter.Get("/summary", app.GetDashboardSummary)
-			dashboardRouter.Get("/config", app.GetAllConfigDetails)
-		})
-
-		uiRouter.Route("/groups", func(groupRouter chi.Router) {
-			groupRouter.Route("/", func(orgSubRouter chi.Router) {
-				groupRouter.With(requirePermission(auth.RoleSuperUser)).Post("/", app.CreateGroup)
-				groupRouter.Get("/", app.GetGroups)
-			})
-
-			groupRouter.Route("/{groupID}", func(groupSubRouter chi.Router) {
-				groupSubRouter.Use(requireGroup(app.groupRepo, app.cache))
-				groupSubRouter.Use(rateLimitByGroupID(app.limiter))
-
-				groupSubRouter.With(requirePermission(auth.RoleUIAdmin)).Get("/", app.GetGroup)
-				groupSubRouter.With(requirePermission(auth.RoleSuperUser)).Put("/", app.UpdateGroup)
-				groupSubRouter.With(requirePermission(auth.RoleSuperUser)).Delete("/", app.DeleteGroup)
-			})
+		uiRouter.Route("/users", func(userRouter chi.Router) {
+			userRouter.Get("/profile", app.GetUser)
+			userRouter.Put("/profile", app.UpdateUser)
+			userRouter.Put("/password", app.UpdatePassword)
+			userRouter.Post("/exists", app.CheckUserExists)
```

Comment:
Some of these are not RESTful

Reply:
They're all scoped under the `users` route. Do you have some suggestions?